tenets/codelingo/go/unused-private-functions/codelingo.yaml

```diff
@@ -0,0 +1,20 @@
+tenets:
+  - name: find-unused-private-funcs
+    doc: Finds unused private functions.
+    flows:
+      codelingo/review:
+        comment: This private function is not used in the package.
+    query: |
+      import codelingo/ast/go
+
+      go.func_decl(depth = any):
+        @review.comment
+        go.ident:
+          private == "true"
+          name as funcName
+          exclude:
+            name == "main"
+      exclude:
+        go.call_expr(depth = any):
+          go.ident:
+            name as funcName
```

Comment:
This should be explicitly scoped to a directory with a `go.dir(depth = any)` fact. Reviews are scoped to the directory that `lingo run review` is run in, so for each private function this will search every subdir to find a call.